docs/pages/includes/database-access/reference/aws-iam/memorydb/access-policy.mdx

```diff
@@ -1,6 +1,41 @@
-{/* this comment is here to make the include below it render */}
+MemoryDB supports IAM authentication for Redis engine version
+7.0 or above.
+This is the recommended way to configure Teleport access to MemoryDB.
 
-(!docs/pages/includes/database-access/reference/aws-iam/redis/access-policy.mdx dbType="MemoryDB" permissionType="memorydb"!)
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "MemoryDBFetchMetadata",
+            "Effect": "Allow",
+            "Action": "memorydb:DescribeClusters",
+            "Resource": "*"
+        },
+        {
+            "Sid": "MemoryDBDescribeUsers",
+            "Effect": "Allow",
+            "Action": "memorydb:DescribeUsers",
+            "Resource": "*"
+        },
+        {
+            "Sid": "MemoryDBConnect",
+            "Effect": "Allow",
+            "Action": "memorydb:Connect",
+            "Resource": [
+                "arn:aws:memorydb:<Var name="us-east-2"/>:<Var name="aws-account-id"/>:cluster:<Var name="cluster-name"/>",
```

Comment:
I'd add instructions to assign a value to this variable above the snippet to avoid the Vale warning

Reply:
i decided to replace the resource here with a `*`, and added a small paragraph below with details on the format of the resource ARN.

this way is more aligned with Redshift and RDS section:
https://goteleport.com/docs/reference/agent-services/database-access-reference/aws/#redshift